src/shared/components/resources/ResourceLink.tsx

```diff
@@ -60,15 +60,15 @@ export default class ResourceLink extends React.Component<
 
     render() {
```

Comment:
Lets get rid of this component.  We need to break it it out into columns anyway so it doesn't really work anymore.  It's pointless.  We just put this out in the Tab